test/hotspot/gtest/testutils.hpp

```diff
@@ -56,4 +56,6 @@ class GtestUtils : public AllStatic {
 
 #define ASSERT_ALIGN(p, n) ASSERT_TRUE(is_aligned(p, n))
 
+#define LOG_HERE(s, ...) { printf(s, __VA_ARGS__); printf("\n"); fflush(stdout); }
+
```

Comment:
In other reviews I've asked the reviewers to use unified logging with the `test` tag. Maybe do the same for this test?

Reply:
Unfortunately, UL is not an option here, since I use it in TEST tests (without _VM, so I may not have VM infrastructure available depending on the order of test execution).